src/attributes.jl

```diff
@@ -1348,6 +1349,14 @@ end
 DualStatus() = DualStatus(1)
 _result_index_field(attr::DualStatus) = attr.N
 
+
+# Cost of bridging constrained variable in S
+struct VariableBridgingCost{S <: AbstractSet} <: AbstractOptimizerAttribute
```

Comment:
If you use `AbstractModelAttribute` here you won't have the issue with `CachingOptimizer`.

Reply:
Thank you for that advice! I changed it.